<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AGMExecutiveClient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <widget class="QWidget" name="lhsParentWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QWidget" name="rhsParentWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Current Model</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Proposal</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRefresh_current_model">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionRefresh_current_model"/>
    <addaction name="actionClear_proposal"/>
    <addaction name="actionSend_proposal"/>
   </widget>
   <addaction name="menuRefresh_current_model"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="toolsWidget">
   <property name="windowTitle">
    <string>      Tools</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="toolsList">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionRefresh_current_model">
   <property name="text">
    <string>Refresh current model</string>
   </property>
  </action>
  <action name="actionClear_proposal">
   <property name="text">
    <string>Clear proposal</string>
   </property>
  </action>
  <action name="actionSend_proposal">
   <property name="text">
    <string>Send proposal</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
